### ISSUE TITLE
User authentication

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ttyborg/castlesand#73]
```
Hello,

I'd like to ask about your opinion on creating user authentication system for 
multiplayer game, so users could register their nickname in order to have 
statistics for games played in the past (i.e. total number of games, wins, 
loses, escapes).

I'm aware it might be a lot of things to take care of in order to enable this 
feature, however I can offer my help as it comes to the programming part. It 
may require having some additional VPS server where the API would be running, 
but I can offer you my own VPS (hosted in France).

Basically what would have to be done is the API (json || xml output), so you 
could implement it in the game and implementing this on the website (i.e. 
profiles.kamremake.com/nickname) would show user's stats.

In the future it might be possible to store data like: stats for most played 
maps, average game time.

Regards,
Lukasz
```

Original issue reported on code.google.com by `luk...@klis.pl` on 13 Mar 2013 at 12:24
